README.md

```diff
@@ -94,9 +94,8 @@ git clone https://github.com/<your-github-id>/geowrangler.git
 cd geowrangler
 virtualenv -p /usr/bin/python3.9 .venv
 source .venv/bin/activate
-pip install pre-commit poetry>=1.2.0
+pip install pre-commit "poetry>=1.2.0"
 pre-commit install
```

Comment:
Let's add a task to cleanup the README.md there is a big chunk talking about setting up pygeos that might no longer be relevant with shapely>=2.0